src/ui/public/filter_editor/filter_field_select.js

```diff
@@ -19,7 +19,7 @@
 
 import 'angular-ui-select';
 import { uiModules } from '../modules';
-import { getFieldOptions } from './lib/filter_editor_utils';
+import { getFilterableFields } from './lib/filter_editor_utils';
```

Comment:
this was renamed as the old name was used for multiple functions, where none of them got you the field options :)